### ISSUE TITLE
Error on print $y *( $z - 2 )

### ISSUE THREAD

**Issue by jidanni** [perltidy/perltidy#62]
```
$ echo 'print $y *( $z - 2 );'|perltidy --noprofile -g; cat perltidy.LOG
## Please see file perltidy.ERR
print $y *( $z - 2 );
perltidy version 20200110 log file on a linux system, OLD_PERL_VERSION=5.032001
Configuration and command line parameters for this run:
-noprofile -g
To find error messages search for 'WARNING' with your editor
Indentation will be with 4 spaces
Line 1 implies starting-indentation-level = 0

########################  WARNING  #########################
1: print $y *( $z - 2 );
            -- ^
found Scalar where operator expected (previous token underlined)
############################################################

########################  WARNING  #########################
There is no previous '(' to match a ')' on line 1
1: print $y *( $z - 2 );
                      ^
############################################################
Suggest including 'use warnings;'
Suggest including 'use strict;'
The nesting depths in the table below are at the start of the lines.
The indicated output line numbers are not always exact.
ci = levels of continuation indentation; bk = 1 if in BLOCK, 0 if not.

in:out indent c b  nesting   code + messages; (messages begin with >>>)
lines  levels i k            (code begins with one '.' per indent level)
------  ----- - - --------   -------------------------------------------
L1:1     i0:0 0 1            print $y *( $z - 2 );
No indentation disagreement seen

No lines exceeded 80 characters
  Maximum output line length was 21 at line 1
```

**Comment by shancock9** [comment]
This was fixed in commit f80d677.

**Comment by shancock9** [comment]
Version 20210625 has been released with a fix for this problem.
